src/pages/gateway/overview.md

```diff
@@ -23,15 +23,14 @@ API Mesh for Adobe Developer App Builder enables developers to integrate a Graph
 
 ## Features
 
--  **API Mesh for Adobe Developer App Builder** - A reverse proxy that accepts most API calls for many backend services
+-  **API Mesh** - A reverse proxy that accepts most API calls for many backend services
 -  **Configurable Integrations** - Low/No-code method to integrate with your own private APIs with Adobe Commerce and other Adobe products, and third-party APIs
-<!-- The two features below likely refer to the App Builder and may need to be deleted before beta -->
 -  **Backwards Compatibility** - Preserve existing APIs while gradually adopting new ones
 -  **Extensibility** - Customize and extend all of your APIs directly in the gateway without making changes to the API source
 
 ## Example use case
 
-Adobe Commerce users can consider API Mesh for Adobe Developer App Builder as a low-code way to integrate other APIs to the Storefront and Storefront Management APIs. It also provides a way for the Storefront API and the Storefront Management APIs to communicate with each other.
+Adobe Commerce users can consider API Mesh as a low-code way to integrate other APIs to the Storefront and Storefront Management APIs. It also provides a way for the Storefront API and the Storefront Management APIs to communicate with each other.
 
 ## Where to go next
```

Comment:
These 2 links 
 -  [Create an Adobe IO account]
 -  [Configure your environment]
are not redirecting properly.